guides/plugins/plugins/content/cms/add-cms-element.md

```diff
@@ -2,468 +2,221 @@
 nav:
   title: Add CMS element
   position: 20
-
 ---
 
-# Add CMS Element
+# Add CMS Elements
+
+## What is an Element?
 
-## Overview
+A CMS element in Shopware is the smallest content unit in the Shopping Experience (CMS) system. Understanding the hierarchy helps clarify what elements are:
 
-This article will teach you how to create a new CMS element via a plugin.
-The plugin in this example will be named `SwagBasicExample`, similar to the other guides.
+### CMS Hierarchy
 
-## Prerequisites
+* Page - The top-level container (e.g., category page, shop page, product page)
+* Section - Horizontal segments within a page (can be single-column or two-column with sidebar)
+* Block - Units that usually span an entire row with custom layout and styling
+* Slots - Named containers within blocks
+* **Elements - The actual content primitives (text, image, video, product listing, etc.) placed inside slots**
```

Comment:
As above.

Maybe it makes sense to add this hierarchy to move this this little section to the parent article, what do you think?